frontend/coprs_frontend/coprs/models.py

```diff
@@ -877,6 +877,13 @@ class Package(db.Model, helpers.Serializer, CoprSearchRelatedData):
     def validate_max_builds(self, field, value):
         return None if value == 0 else value
 
+    # max package build time
+    timeout = db.Column(db.Integer, index=True)
```

Comment:
I don't think we want the index. There is no sorting or searching based on the timeout.

Reply:
Done.